### ISSUE TITLE
[Feature Request] Skills extra bonus for dice rolls only

### ISSUE THREAD

**Issue by dacancino** [anvil-vtt/FateX#71]
![image](https://user-images.githubusercontent.com/46689291/123563215-6a5a7c00-d781-11eb-8a9f-23138ddee608.png)

Hi! Thanks for your great work with this System. I'm designing my own set of rules and Ambientation based on Fate Condensed.

Something i'd really like to see implemented, like the image i provide here, is a secondary bonus number for the skills, which only affects the roll dice, but not the skills order. Sometimes one character has a special weapon or equipment that adds a bonus to a certain skill, but doesn't affecting the "skill pyramid".

I'd love if you can implement something like that... Maybe as an activatable box in the caracter's skill.
Have a good day!

**Comment by PatrickBauer** [comment]
@dacancino Thank you for this request: Would that bonus be applied every time? Is it like a flat bonus for every roll on this skill for this user?

**Comment by dacancino** [comment]
> @dacancino Thank you for this request: Would that bonus be applied every time? Is it like a flat bonus for every roll on this skill for this user?

@PatrickBauer Yes! Like a flat bonus for every roll, until the character's owner or the GM decide to deactivate it =)

**Comment by Pjb518** [comment]
@dacancino Adding a static modifier field to skills is pretty straightforward, but I'm not sure I'd want the interface cluttered up with all these additional bonuses.

Perhaps we can denote modified skill values with an asterisk or a colour difference to make it easy to keep track of skills which have been granted a bonus.

Would that work for you? Or is it really important that you can see these bonuses at all times?

**Comment by dacancino** [comment]
> @dacancino Adding a static modifier field to skills is pretty straightforward, but I'm not sure I'd want the interface cluttered up with all these additional bonuses.
> 
> Perhaps we can denote modified skill values with an asterisk or a colour difference to make it easy to keep track of skills which have been granted a bonus.
> 
> Would that work for you? Or is it really important that you can see these bonuses at all times?

@Pjb518 Hi!! I just posted the image to show the concept c: But sure! Could be something less invasive.
Maybe a different tab in the sheet where the GM or the player could set this flat bonuses...  it is important that the bonus could be any number between 1 and 9.

Thanks for the reply =)
